tests/org.eclipse.passage.lic.jface.tests/OSGI-INF/l10n/bundle.properties

```diff
@@ -14,7 +14,7 @@
 
 Bundle-Name = Passage LIC JFace Tests
 Bundle-Vendor = Eclipse Passage
-Bundle-Copyright = Copyright (c) 2019 ArSysOp and others.\n\
+Bundle-Copyright = Copyright (c) 2019, 2020 ArSysOp and others.\n\
 \n\
 This program and the accompanying materials are made\n\
 available under the terms of the Eclipse Public License 2.0\n\

```

Comment:
There are a strange symbol at the ends of four rows. Can we get rid of it?

Reply:
I'll try

Reply:
Looked through HEX - it has normal 0A0D sequence here, no idea what it is